variables.py

```diff
@@ -62,38 +66,40 @@
 # **** Challenge 2: Problem 1 ****
 # Store the sum of num1 and num2 in a variable.
 # Print the sum.
-
+print(5 + 10)
 
 # **** Challenge 2: Problem 2 ****
 # Store the difference between sum (above) and 7 in a variable.
 # Print the difference.
-
+print(15 - 7)
 
 # **** Challenge 2: Problem 3 ****
 # Store the product of the difference and 3 in a variable.
 # Print the product.
+print(8 * 3)
 
 
 # **** Challenge 2: Problem 4 ****
 # Store the power of the product squared in a variable.
 # Print the power.
-
+print(24 ** 2)
 
 # **** Challenge 2: Problem 5 ****
 # Store the quotient of the power divided by 4 in a variable.
 # Print the quotient.
-
+print(576/4)
 
 # **** Challenge 2: Problem 6 ****
 # Store the remainder of the quotient divided by 2 in a variable.
 # Print the remainder.
-
+print(576%4)
 
 # **** Upchallenge! ****
 # Given a variable that stores a number, print the values that come right before and right after.
 # For example, given the number 4, the program should print 3 and 5. 
 # Your code should work even if the value of the variable is changed.
-
-num = 9
+num = 3
+print({num - 1})
+print({num + 1})
```

Comment:
This is great! The curly braces ( {} ) are not necessary in this case. You only need the braces when you are trying to insert a variable in a string. 

```suggestion
print(num - 1)
print(num + 1)
```